package.json

```diff
@@ -29,6 +29,7 @@
     "@dnd-kit/sortable": "^7.0.2",
     "@dnd-kit/utilities": "^3.2.1",
     "@juggle/resize-observer": "^3.3.1",
+    "@tanstack/react-virtual": "^3.0.1",
```

Comment:
Should we pin this dependency to avoid unexpected versions installed by our consumers?

Reply:
We don't pin other dependencies, why should we pin this one?

Reply:
This used to be pinned before because of bugs. Check the pre-github commit history

Reply:
What is your final decision here? How about asking the team opinion?

Reply:
Agreed on having this dependency pinned.